<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HamiltonianHeisenbergNeighboursWidget</class>
 <widget class="QWidget" name="HamiltonianHeisenbergNeighboursWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>711</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="8" column="2">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="2">
        <layout class="QGridLayout" name="gridLayout_31">
         <item row="0" column="0">
          <widget class="QLabel" name="label_23">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Apply to</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_Hamiltonian_Iso_ApplyTo">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Current Image</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Image Chain</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All Images</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="9" column="0">
          <layout class="QGridLayout" name="gridLayout_23">
           <item row="0" column="1">
            <widget class="QLabel" name="label_17">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Direction (x,y,z)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_14">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBox_extH">
           <property name="text">
            <string>External Field</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="checkBox_aniso">
           <property name="text">
            <string>Anisotropy</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_iso_periodical_c">
           <property name="text">
            <string>Periodical c</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_iso_periodical_a">
           <property name="text">
            <string>Periodical a</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_iso_periodical_b">
           <property name="text">
            <string>Periodical b</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="0">
          <widget class="QCheckBox" name="checkBox_exchange">
           <property name="text">
            <string>Exchange</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_extHy">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_extHx">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_extHz">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_muSpin">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <layout class="QGridLayout" name="gridLayout_25">
           <item row="1" column="0">
            <spacer name="horizontalSpacer_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_14">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Direction (x,y,z)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>mu_spin</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="19" column="0">
          <widget class="QCheckBox" name="checkBox_ddi">
           <property name="text">
            <string>Dipole-Dipole radius</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_extH">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="20" column="0">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="1">
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_anisoz">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_anisoy">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_anisox">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="8" column="1">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="19" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_ddi_radius">
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QCheckBox" name="checkBox_dmi">
           <property name="text">
            <string>DMI</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="lineEdit_bqe">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <layout class="QGridLayout" name="gridLayout_dmi">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Number of Shells</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_dmi_nshells"/>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="16" column="0">
          <widget class="QCheckBox" name="checkBox_fourspin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>FourSpin Magnitude</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QCheckBox" name="checkBox_bqe">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>BQE Magnitude</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="18" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="16" column="1">
          <widget class="QLineEdit" name="lineEdit_fourspin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <layout class="QGridLayout" name="gridLayout_exchange">
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_exchange_nshells"/>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Number of Shells</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="12" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>comboBox_Hamiltonian_Iso_ApplyTo</tabstop>
  <tabstop>checkBox_iso_periodical_a</tabstop>
  <tabstop>checkBox_iso_periodical_b</tabstop>
  <tabstop>checkBox_iso_periodical_c</tabstop>
  <tabstop>checkBox_extH</tabstop>
  <tabstop>lineEdit_extH</tabstop>
  <tabstop>lineEdit_muSpin</tabstop>
  <tabstop>lineEdit_extHx</tabstop>
  <tabstop>lineEdit_extHy</tabstop>
  <tabstop>lineEdit_extHz</tabstop>
  <tabstop>checkBox_aniso</tabstop>
  <tabstop>lineEdit_aniso</tabstop>
  <tabstop>lineEdit_anisox</tabstop>
  <tabstop>lineEdit_anisoy</tabstop>
  <tabstop>lineEdit_anisoz</tabstop>
  <tabstop>checkBox_exchange</tabstop>
  <tabstop>spinBox_exchange_nshells</tabstop>
  <tabstop>checkBox_dmi</tabstop>
  <tabstop>spinBox_dmi_nshells</tabstop>
  <tabstop>checkBox_bqe</tabstop>
  <tabstop>lineEdit_bqe</tabstop>
  <tabstop>checkBox_fourspin</tabstop>
  <tabstop>lineEdit_fourspin</tabstop>
  <tabstop>checkBox_ddi</tabstop>
  <tabstop>doubleSpinBox_ddi_radius</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
